### ISSUE TITLE
Support for smtps:// MAIL_URL

### ISSUE THREAD

**Issue by MichaelJCole** [meteor/meteor#7042]
Ok, couple of things here:

1) Meteor uses port 465 to indicate secure connection.  [Port 465 was never a standard port and was assigned to something else](http://blog.mailgun.com/25-465-587-what-port-should-i-use/).

2) I'm getting an email transport error similar to this one: [Greeting not received]( https://github.com/nodemailer/nodemailer/issues/441).   Related to #5284

My email provider (Sparkpost is the new Mandrill) wants to use STARTTLS on port 587.  I don't see a way to configure that in Meteor, we just have MAIL_URL.

"smtps" appears to be a common protocol, ([see first example](https://github.com/nodemailer/nodemailer)), but Meteor doesn't allow it ([email code](https://github.com/meteor/meteor/blob/devel/packages/email/email.js#L20-L36)), and only uses secureConnection if the port is 465. 

This seems like a bug, but maybe Meteor makes this choice for a reason?  What's the best way to configure Meteor to use STARTTLS on port 587?

Attached PR sets secure connection if 
```
{ secureConnection: (port === 465) || (mailUrl.protocol === 'smtps:'),
```

This would be a non-breaking change, that is more standards based.

Thanks!

Mike

**Comment by abernix** [comment]
My initial impression is that there is something missing in your PR.   The code snippet you put above:
```
{ secureConnection: (port === 465) || (mailUrl.protocol === 'smtps:'),
```
...is not in the PR.  Will comment more in the PR.

**Comment by abernix** [comment]
Closing as this should have been resolved with PR #7043 which was merged and should be in Meteor as of 1.3.3.1.
